src/AdvApi32.Desktop/AdvApi32.cs

```diff
@@ -507,10 +552,10 @@ public static partial class AdvApi32
         /// </returns>
         [DllImport(api_ms_win_security_base_l1_2_0, SetLastError = true)]
         [return: MarshalAs(UnmanagedType.Bool)]
-        public static extern unsafe bool GetTokenInformation(
+        public static extern bool GetTokenInformation(
             SafeObjectHandle TokenHandle,
             TOKEN_INFORMATION_CLASS TokenInformationClass,
-            void* TokenInformation,
+            IntPtr TokenInformation,
```

Comment:
@vbfox , with the generated code, isn't both ways now present for the end-user? I think he might even not notice the change because the Code Generator puts the IntPtr alternative during compilation, right?

I was thinking mostly on normalizing and making the patterns match on this kind of functions.